ava-labs/core-mobile
# Fix exit logic for Android

## Description

I needed  action to lock the app immediately (e.g. seedles token expired) and then noticed that current exit logic for andoird doesn't work well with new seedless flow.
This PR adds that feature and fixes said problem.

## Screenshots/Videos


https://github.com/ava-labs/avalanche-wallet-apps/assets/6573904/0bb26ae5-b996-432a-833a-0e205e3897e5


## Testing
- press back button in Android
- confirm with pressing Exit on dialog
- start App agian
- app should open pin/biometry screen

## Checklist

Please check all that apply (if applicable)
